src/cljs/main/broadfcui/user_info.cljs

```diff
@@ -2,6 +2,7 @@
   (:require
    [broadfcui.common :as common]
    [broadfcui.endpoints :as endpoints]
+   [broadfcui.utils :as utils]
```

Comment:
necessary?

Reply:
:man_shrugging: I added it to debug, and we like to leave them so you can always log without worrying about imports.

Reply:
all good, I don't mind